airflow/providers/google/cloud/operators/dataflow.py

```diff
@@ -1358,3 +1359,236 @@ def execute(self, context: Context) -> None:
             self.log.info("No jobs to stop")
 
         return None
+
+
+class DataflowCreatePipelineOperator(GoogleCloudBaseOperator):
```

Comment:
I think it's better to name it DataflowCreate**Data**PipelineOperator to comply with the API's name (same goes for the other operators).
I'm aware of the wordiness, but it could help avoid confusion with regular Dataflow jobs operators. What do you think?

Reply:
Thank you for checking my changes :)
Current PR was created to get rid of exactly this DataPipeline as a separate service inside Google since this is a part of Dataflow itself. Right now in all the operators we have both in Dataflow and Apache Beam the Pipeline object refers to exactly the Pipeline. Currently all the operators skip this step with creating separate Pipeline object but take its configuration and run it to create a Job. 
In my understanding changing this name here in this PR will lead to countless changes in other operators 🥲